.travis.yml

```diff
@@ -1,31 +1,35 @@
 language: python
-branches:
-    only:
-        - master
-        - dogfood
-services:
-  - postgres
 python:
   - "2.7"
+
+virtualenv:
+  system_site_packages: true
+
+sudo: required
+dist: trusty
+
+cache:
+  - pip
+
+addons:
+  postgresql: "9.3"
```

Comment:
We're not using any specific feature that's available only in one of these postgresql releases, but by default travis-ci uses 9.1, while ubuntu trusty ships with 9.3. I felt a bit safer as an integration test to pair trusty with 9.3.